packages/sources/paypal/src/adapter.ts

```diff
@@ -0,0 +1,42 @@
+import { Requester, Validator, AdapterError } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, ExecuteFactory } from '@chainlink/types'
+import { makeConfig, DEFAULT_ENDPOINT } from './config'
+import { sendPayout, getPayout } from './endpoint'
+
+const inputParams = {
+  endpoint: false,
+}
+
+const paramOptions = {
+  endpoint: ['getpayout', 'sendpayout'],
```

Comment:
Maybe we should call these something a little bit more generic. 

See [this comment](https://github.com/smartcontractkit/external-adapters-js/pull/417#discussion_r609118508) for a bit more context.

We haven't fully fleshed out this pattern, but one path forward I could see is:
```suggestion
  endpoint: [getpayout.NAME, sendpayout,NAME, 'read', 'write'],
```
We still want to use allow `getpayout` to stay close to the API, but we also want to be able to use a unified API endpoint if we envision more adapters getting made with similar functionality (think: Cash App adapter, Venmo adapter). Use case: I'm an artist and I want a smart contract that will send an NFT once paid across one of my various money apps.

Rodrigo & Evangelos have examples where he put both of these endpoint names in the endpoints `NAME` itself (as an Array).

I think we should move toward keeping this adapter un-aware of the unified `endpoint` and instead map them in another package or middleware -- but this is out of scope for just getting this in. Just something to have in your mind.